sbin/prepareWorkspace.sh

```diff
@@ -492,44 +492,13 @@ checkingAndDownloadingFreeType() {
   fi
 }
 
-# Download our security certificates
-downloadCerts() {
-  local caLink="$1"
-
-  mkdir -p "security"
-  # Temporary fudge as curl on my windows boxes is exiting with RC=127
-  if [[ "$OSTYPE" == "cygwin" ]] || [[ "$OSTYPE" == "msys" ]]; then
-    wget -O "./security/cacerts" "${caLink}"
-  else
-    curl -L -o "./security/cacerts" "${caLink}"
-  fi
+# Generates cacerts file
+prepareCacerts() {
```

Comment:
Sorry for being a pain with another change but can we add something into this function that puts a message including "cacerts" to the log to make it easy to identify the correct section (previous messages about this all had it to make it easy to jump to the log) ... Just something like:

`echo "Generating cacerts from Mozilla's bundle"`

Then I'll be happy with it

Reply:
Done